pkg/cloudmap/client.go

```diff
@@ -112,20 +112,13 @@ func (sdc *serviceDiscoveryClient) CreateService(ctx context.Context, nsName str
 
 func (sdc *serviceDiscoveryClient) GetService(ctx context.Context, nsName string, svcName string) (svc *model.Service, err error) {
 	sdc.log.Info("fetching a service", "namespace", nsName, "name", svcName)
-	if endpts, found := sdc.cache.GetEndpoints(nsName, svcName); found {
```

Comment:
Why is this cache call removed?

Reply:
Discussed offline, cache call was removed because it still contained endpoints which were not part of current cluster but part of cluster set. 
Solution: Since we are not dealing with huge services (max size 1000), we will still return cached endpoints filtered on `endpts.CLUSTER_ID` attribute